test/TestApps/ToastNotificationsTestApp/ToastNotificationsTestApp.vcxproj

```diff
@@ -196,36 +183,26 @@
   <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
     <ClCompile>
       <PrecompiledHeader>Use</PrecompiledHeader>
-      <WarningLevel>Level3</WarningLevel>
-      <SDLCheck>true</SDLCheck>
       <AdditionalIncludeDirectories>%(AdditionalIncludeDirectories);$(OutDir)\..\WindowsAppRuntime_DLL;$(OutDir)\..\WindowsAppRuntime_BootstrapDLL;..\..\inc</AdditionalIncludeDirectories>
       <PreprocessorDefinitions>_DEBUG;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <PrecompiledHeaderFile>pch.h</PrecompiledHeaderFile>
-      <LanguageStandard>stdcpp17</LanguageStandard>
     </ClCompile>
     <Link>
       <SubSystem>Console</SubSystem>
       <AdditionalLibraryDirectories>%(AdditionalLibraryDirectories);$(OutDir)\..\WindowsAppRuntime_DLL</AdditionalLibraryDirectories>
       <AdditionalDependencies>onecore.lib;onecoreuap.lib;Microsoft.WindowsAppRuntime.lib;%(AdditionalDependencies)</AdditionalDependencies>
-      <DelayLoadDLLs>Microsoft.WindowsAppRuntime.Bootstrap.dll</DelayLoadDLLs>
+      <DelayLoadDLLs>Microsoft.WindowsAppRuntime.Bootstrap.dll;%(DelayLoadDLLs)</DelayLoadDLLs>
```

Comment:
>DelayLoadDLLs [](http://example.com/codeflow?start=65&length=13)

Good find.  Missing delayloads is bad for perf.

Reply:
It's only a test project but yes, missing could be bad.

Worse IMO are the inconsistencies across Configration+Platform. We have 6 permutations. I've found several that only set options for 5, or 1, or somewhere in between. Fixed that up, for any options I messed with. May still have some inconsistencies in other properties.

I've had 'sanitize projects' in my wishlist for ages, and incrementally get closer. There's more to do but only so many hours in a do, so today is better than yesterday, and tomorrow will be better still :-) Just so long as nothing egregious pops up we can get to it incrementally as time permits.